lib/util/exec/docker/index.ts

```diff
@@ -117,17 +117,17 @@ export async function getDockerTag(
     const version = versions.sort(ver.sortVersions.bind(ver)).pop();
     if (version) {
       logger.debug(
-        { depName, scheme, constraint, version },
+        { depName: packageName, scheme, constraint, version },
```

Comment:
```suggestion
        { packageName, scheme, constraint, version },
```
Why not simply log this as `packageName`? If we use `depName` here it could be confusing.